README.md

```diff
@@ -1,2 +1,48 @@
-# metagenomome_assembly
+# Metagenome_assembly
 WDL Workflow for metagenome assembly
+
+## Introduction
+### This pipeline will perform;
+* Pre-processing of reads with Trim Galore and Kneaddata
+* Metagenomics assembly with Megahit
+* Gene prediction
+* Mapping of reads against the contigs 
+* Metagenome binning using  MetaBAT2 
```

Comment:
there is a double-space before MetaBAT2.
sorry, double spaces are my pet peeve... could you remove all?

Reply:
Okay